vagrant/Vagrantfile

```diff
@@ -44,13 +45,16 @@ Vagrant.configure(2) do |config|
     aws.session_token = ENV['AWS_SESSION_TOKEN']
     aws.associate_public_ip = true
     aws.tags = { 'Name' => (ENV['DEPLOY_ENV'] || ENV['USER']) + " concourse" }
+    aws.keypair_name = ENV['VAGRANT_SSH_KEY_NAME']
```

Comment:
Is the addition of a separate SSH key required? If so, could you explain why in the commit message?